src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM_LENGHT = 20;

```

Comment:
previous comment not fixed
please see hidden comments on Github and fix everything